build.gradle

```diff
@@ -146,19 +136,6 @@ jar {
 			}
 		}
 	}
-
-	into ('mods/railcraft/api') {
```

Comment:
Removing this (and also removing railcraft from sourceSet) eventually removed all the railcraft api classes from build output. Since railcraft doesn't distribute an api jar like CodeChickenLib does, class loader will fail to find railcraft api classes.

Reply:
1.7.10 API isn’t available in our repo or theirs. It’s currently a bundled lib which I just need to move over. I’m aware of it missing from the final build along with some assets, which must also be missing from our RC repo.

Reply:
RF API is gone (somehow, probably repo renamed) but the railcraft API is intact. The link on github is clickable and I can also clone it as submodule just fine.

Reply:
There’s no 1.7.10 branch from what I saw.